frontend/src/tests/components/Users/UsersTable.test.js

```diff
@@ -1,12 +1,26 @@
-import { render } from "@testing-library/react";
+import React from 'react';
+import { fireEvent, render, act} from "@testing-library/react";
 import usersFixtures from "fixtures/usersFixtures";
 import UsersTable from "main/components/Users/UsersTable"
+
 import { QueryClient, QueryClientProvider } from "react-query";
 
+const mockedNavigate = jest.fn();
+
+// jest.mock('react-router-dom', () => ({
+//     ...jest.requireActual('react-router-dom'),
+//     useNavigate: () => mockedNavigate
+// }));
```

Comment:
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed